src/main/ncdio_pio.F90.in

```diff
@@ -1291,7 +1291,8 @@ contains
        else
           idata1d(1) = 0
        end if
-       status = pio_put_var(ncid, varid, start(1:1), count(1:1), idata1d)
+!       status = pio_put_var(ncid, varid, start(1:1), count(1:1), idata1d)
```

Comment:
This line should be removed rather than commented out. I can do that with your permission, or you can do it.